internal/ethapi/api.go

```diff
@@ -1093,14 +1094,23 @@ func (args *SendTxArgs) setDefaults(ctx context.Context, b Backend) error {
 		}
 		args.Nonce = (*hexutil.Uint64)(&nonce)
 	}
+	if args.Input != nil && args.Data != nil {
+		return errors.New(`"data" and "input" are mutually exclusive`)
```

Comment:
The error message is a bit obfuscated. It's not entirely obvious from the error message that it will fail if the user supplies both. When two entities are mutually exclusive, it does not require that one be nil.

Perhaps it makes sense to have the error message something on the lines of: "Please supply either data or input."